test/test_tree_2d_euler.jl

```diff
@@ -384,7 +385,18 @@ end
                         ],
                         tspan=(0.0, 1.0),
                         initial_refinement_level=4,
-                        coverage_override=(maxiters = 6,))
+                        coverage_override=(maxiters = 6,),
+                        save_errors=true,
+                        output_directory="out")
+    lines = readlines("out/deviations.txt")
+    @test lines[1] == "# iter, simu_time, rho_min, rho_max"
+    cmd = string(Base.julia_cmd())
+    coverage = occursin("--code-coverage", cmd) &&
+               !occursin("--code-coverage=none", cmd)
+    if !coverage
+        # Run without coverage takes 96 time steps.
+        @test startswith(lines[end], "96")
+    end
```

Comment:
I don't have a strong opinion on this. Though, checking whether coverage is enabled is not really nice. Alternatively, I could activate the test for the case with coverage as well, but then I would need to write something like
`@test startswith(lines[end], "96") || startswith(lines[end], "1")`

Reply:
Can't you just check the first line for some additional values besides the time step coutner?

Reply:
```suggestion
    cmd = string(Base.julia_cmd())
    coverage = occursin("--code-coverage", cmd) &&
               !occursin("--code-coverage=none", cmd)
    if coverage
        # Run with coverage takes 1 time step.
        @test startswith(lines[end], "1")
    else
        # Run without coverage takes 96 time steps.
        @test startswith(lines[end], "96")
    end
```
Why not at least check both?

Reply:
Yes that's better. Added it in [b268f2f](https://github.com/trixi-framework/Trixi.jl/pull/1824/commits/b268f2f2467b0ebac6d918129cf72c0479767973).